content/applications/finance/fiscal_localizations/united_kingdom.rst

```diff
@@ -27,17 +27,28 @@ Reports` modules to get all the features of the UK localization.
         - Allows sending the tax report via the MTD-VAT API to HMRC.
    * - :guilabel:`UK BACS Payment Files`
      - `l10n_uk_bacs`
-     - Allows generating :ref:`united-kingdom/BACS-files` for bill and invoice payments
+     - Allows generating :ref:`localization/united-kingdom/BACS-files` for bill and invoice payments
+   * - :guilabel:`UK - Construction Industry Scheme`
+     - `l10n_uk_reports_cis`
+     -  - Allows sending the Monthly return to |HMRC|
+        - CIS Deduction (GB) report for UK construction industry
+   * - :guilabel:`UK - HMRC API`
+     - `l10n_uk_hmrc`
+     -  - Includes the basics of |HMRC|.
```

Comment:
```suggestion
     -  Includes the basics of |HMRC|.
```
isn't this CIS-specific (not sure what else the "basics of HRMC" entail but we might want to play it safe here? I would also add a link to that section in the doc